docs/src/content/docs/fr/guides/i18n.mdx

```diff
@@ -142,13 +146,17 @@ Si une traduction n'est pas encore disponible pour une langue, Starlight affiche
 ## Traduire l'interface utilisateur de Starlight
 
 import LanguagesList from '~/components/languages-list.astro';
+import UIStringsList from '~/components/ui-strings-list.astro';
 
 En plus d'héberger des fichiers de contenu traduits, Starlight vous permet de traduire les chaînes de l'interface utilisateur par défaut (par exemple, l'en-tête "Sur cette page" dans la table des matières) afin que vos lecteurs puissent découvrir votre site entièrement dans la langue sélectionnée.
 
-Les textes de l'interface utilisateur traduits en <LanguagesList /> sont disponibles prêts à l'emploi, et nous acceptons les [contributions pour ajouter d'autres langues par défaut](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md).
+<LanguagesList startsSentence /> les chaînes traduites de l'UI sont livrées de base,
+et nous accueillons volontiers [les contributions visant à ajouter d'autres langues par défaut](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md).
```

Comment:
I'm a bit confused about this one? I guess this should be reverted?

Reply:
Or maybe 
> les chaînes traduites de l'UI sont livrées de base,
et nous acceptons [les contributions pour ajouter d'autres langues par défaut](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md).

Reply:
I think my question is more about why was this changed in the first place? Is there an issue with the current translation? 

> Les textes de l’interface utilisateur traduits en allemand, anglais, arabe, chinois, chinois (Taïwan), coréen, danois, espagnol, français, galicien, hébreu, hindi, indonésien, italien, japonais, néerlandais, norvégien bokmål, persan, polonais, portugais, roumain, russe, suédois, tchèque, turc, ukrainien et vietnamien sont disponibles prêts à l’emploi, et nous acceptons les [contributions pour ajouter d’autres langues par défaut](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md).

To me, starting with the list of languages does not seem right?

Reply:
Oh, or maybe you're trying to match the English version which starts by `<LanguagesList startsSentence />`? If this is it, I see the confusion.

The component is meant to be used differently between languages so that the list of languages makes sense in the current locale. Having a 1:1 structure match with the English version is not a goal.

If that was the goal, we can revert the change so that theses lines are not modified in this PR 👍

Reply:
Because in the original file at L153,  the line was changed with `<LanguagesList startsSentence /> at the starting
https://github.com/withastro/starlight/blob/554c670c402c5cd87fe376d506292a8f0b017ed2/docs/src/content/docs/guides/i18n.mdx#L153

Reply:
Oooooooh... I'm dumb.. yeah you right, and translated without know the content of `LanguagesList` 🫢

Reply:
> Oh, or maybe you're trying to match the English version which starts by `<LanguagesList startsSentence />`? If this is it, I see the confusion.
> 
> The component is meant to be used differently between languages so that the list of languages makes sense in the current locale. Having a 1:1 structure match with the English version is not a goal.
> 
> If that was the goal, we can revert the change so that theses lines are not modified in this PR 👍

yeah I read that just now, nosense my translation here xD
I edit that

Reply:
it's gone back!